app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/github/ContributorsData.kt

```diff
@@ -0,0 +1,39 @@
+package pl.szczodrzynski.edziennik.data.api.edziennik.github
+
+import com.google.gson.GsonBuilder
+import okhttp3.*
+import pl.szczodrzynski.edziennik.data.api.edziennik.github.Contributor
+import pl.szczodrzynski.edziennik.App
+import java.io.IOException
+import kotlin.coroutines.suspendCoroutine
+
+object ContributorsData {
+
+    suspend fun getContributors(app: App): List<Contributor> {
```

Comment:
Instead of requesting the list from the API every time, you can "cache" the list in this `object` - make a nullable variable. Then, when the variable is already a populated list, return it instead.